CHANGELOG.md

```diff
@@ -5,6 +5,21 @@ on [Keep a CHANGELOG](http://keepachangelog.com/). This project adheres to
 
 ## [Un-released]
 
+## [1.3.0] - 2016-09-26
```

Comment:
All these changes should be under the heading [Unreleased] until we do a release. We're currently on1.2.0-SNAPSHOT, so the next release will be 1.2.0. But we don't update this file until we create the release tag.